docs/proposals/proposal-003-report.md

```diff
@@ -211,31 +213,166 @@ As already mentioned we will have two sets of metrics:
 For the project related one Falco has already requested this metrics:
 
 ```
-rate(container_cpu_usage_seconds_total[5m])
+container_cpu_usage_seconds_total
 container_memory_rss
 container_memory_working_set_bytes
 ```
 
-For the Sustainbility metrics we will keep this one:
+These metrics are provided by cAdvisor and scraped from the kubelet.
+
+For the Sustainability metrics we will keep this one:
 
 `kepler_container_joules_total`
 
+### Metadata
+
+As well as the metrics we will collect additional metadata about the pipeline run. This is to provide context on how the results were measured and to allow results to be compared over time.
+
+This metadata will include at least these attributes and more may be added over time.
+
+- Project name
+- Project version (which release was benchmarked)
+- Datetime (when the benchmark was performed)
+- Project configuration (which configuration was benchmarked)
+- Additional info
+  - Pipeline version
+  - Kubernetes version
+  - Kepler version
+  - Equinix Metal instance type
+
 ### Collect
 
-TBD
+The Green Reviews pipeline collects the metrics values for the pipeline run so they can be stored. 
+
+**Note:** 2 options are presented for comparision. The proposal will be updated once an option has been selected.
+
+#### Query Prometheus
+
+Query Prometheus directly by running PromQL queries.
+
+**Strengths**
+
+- All metrics are available in Prometheus.
+- PromQL queries can be executed against the Prometheus serve.
+
+**Weaknesses**
+
+- Dependency on having Prometheus installed in the cluster.
+- Executing PromQL queries may need higher level tooling like the Prometheus Go SDK.
+
+#### Scrape the metrics endpoint directly
+
+Collect metrics by making curl requests to the `/metrics` endpoints.
 
-A prometheus query?
-A direct curl to the sources?
+**Strengths**
 
-Evaluate pros and cons
+- Simple.
+- Minimal dependencies just needs curl.
+
+**Weaknesses**
+
+- Need to scrape both kepler and kubelet metrics endpoints.
+- No ability to make PromQL queries.
```

Comment:
> Not sure exactly what you mean with this one. Is it that you can't get for example averages and stuff like that?

Yes if we scrape the endpoint we can only access the raw values and would have to do the calculations ourselves.

> Actually this could be a dealbreaker, for example if we need the energy consumption from x to y timestamp (the time when the benchmarking ran), we can't do that by just curling the /metrics endpoints, right?

Yes I think it could be a dealbreaker. By using PromQL we can leverage Prometheus to help with the more complex parts of the calculation.